services/121-service/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16-slim as base
+FROM node:18-slim as base
```

Comment:
I don't think we want to mix in this big change in with this PR, or is it a blocking issue for the back-end changes?

Reply:
This is just for testing, will remove it when everything is ready. 
And this PR is not ready for review yet, because started to add front end side to it as requested to do it together with backend. Will ping you when it is ready  :)